docs/import-from-github.mdx

```diff
@@ -1,54 +1,80 @@
 ---
-title: Import from GitHub
-description: 'Import your machines from GitHub into Stately Studio.'
+title: Connect GitHub repo
+description: 'Sync your machines from GitHub with Stately Studio.'
 ---
 
-There are currently two ways to import your machines from GitHub into Stately Studio:
-
-1. [Import all machines from a GitHub repository](#import-all-machines-from-a-github-repository).
-2. [Import machines from a GitHub URL.](#import-machine-from-a-github-url)
+You can connect a GitHub repo to a new project in Stately Studio, keeping updates between GitHub and Stately Studio in sync. Connecting a GitHub repo allows you to import your existing machines from GitHub and push changes to your machines back to your repo as pull requests.
 
 :::studio
 
-Import from GitHub is a [premium feature](/studio-pro-plan.mdx) of Stately Studio.
-
-We offer a **free trial** on the [Stately Studio Pro account](https://stately.ai/pricing) so you can explore how our Pro features work for you and your team.
+Connect GitHub repo is a premium feature of Stately Studio. You can try Stately Studio’s premium plans with a free trial. [Check out the features on our Pro plan](studio-pro-plan.mdx), [Team plan](studio-team-plan.mdx), [Enterprise plan](studio-enterprise-plan.mdx) or [upgrade your existing plan](https://stately.ai/registry/billing).
```

Comment:
Adding this comment here because this paragraph links to the Stately Studio Pro Plan page. Should we also include updating "GitHub Sync (coming soon!)" by removing the "(coming soon!)"?

Reply:
And I guess that Pro feature bullet point can then link back to this page as well.

Reply:
Yes, thanks! I’ll do this.